### PR TITLE
Refactor and minor fixes

### PR DESCRIPTION
**Pull request by daanraman**

### CODE REVIEW

File: app/helpers/analyzer.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 import dateutil
 import copy
+import numpy as np
```
Comment (detobel36) [inline app/helpers/analyzer.py]: Why adding numpy here ?

File: app/outliers.py [hunk 2]
```diff
@@ -191,7 +116,7 @@ def print_analysis_summary(analyzed_models):
                 should_schedule_next_run = True
 
             # Check for configuration file changes and load them in case it's needed
-            if len(file_mod_watcher.files_changed()) > 0:
+            if file_mod_watcher.files_changed():
```
Comment (detobel36) [inline app/outliers.py]: Here the condition will be done with a number (`files_changed` return the number of file that have changes)

File: app/outliers.py [hunk 3]
```diff
@@ -231,7 +156,7 @@ def print_analysis_summary(analyzed_models):
         errored_models = [analyzer for analyzer in analyzed_models if analyzer.unknown_error_analysis]
 
         # Check the result of the analysis
-        if len(errored_models) > 0:
+        if errored_models:
```
Comment (detobel36) [inline app/outliers.py]: Same that `files_changed`, here `errored_models` is a list. So we based on the fact that an empty list is `False`

File: app/outliers.py [hunk 4]
```diff
@@ -23,145 +22,71 @@
...
+    if settings.args.run_mode == "daemon":
+        run_daemon_mode()
 
-    for index, analyzer in enumerate(analyzers_to_evaluate):
-        try:
-            analyzer.analysis_start_time = datetime.today().timestamp()
-            analyzer.evaluate_model()
-            analyzer.analysis_end_time = datetime.today().timestamp()
-            analyzer.completed_analysis = True
+    if settings.args.run_mode == "interactive":
```
Comment (detobel36) [inline app/outliers.py]: Possibility to transform it into an `elif`

**Comment by daanraman** [comment]
I used it to test something, can be removed

[cid:image006.png@01D54135.FFB064C0]
Daan Raman
NVISO. Security. Research. Risk.
M: +32 478 65 79 36 ▪ E: draman@nviso.be
www.nviso.be<https://www.nviso.be/> ▪ PGP Key ID: F648BD9C
[cid:image007.png@01D54135.FFB064C0]<https://twitter.com/NVISOsecurity>[cid:image008.png@01D54135.FFB064C0]<https://linkedin.com/company/nviso-cyber>[cid:image009.png@01D54135.FFB064C0]<https://twitter.com/nviso_labs>[cid:image010.png@01D54135.FFB064C0]<https://blog.nviso.be/>

**Pull request merged**